File: Backend/Models/Lokacija.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    [Table("lokacija")]
    public class Lokacija
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Required]
        [Column("pavadinimas")]
        public string Pavadinimas { get; set; } = null!;
    }
}

File: frontend/lib/models/lokacija.dart
class Lokacija {
  final int id;
  final String pavadinimas;

  Lokacija({required this.id, required this.pavadinimas});

  factory Lokacija.fromJson(Map<String, dynamic> j) => Lokacija(
        id: j['id'] as int,
        pavadinimas: (j['pavadinimas'] ?? j['Pavadinimas'] ?? '') as String,
      );
}
